app/components/publications/publication/proofs/proof-requested-panel.js

```diff
@@ -0,0 +1,44 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { isEmpty } from '@ember/utils';
+import { task } from 'ember-concurrency-decorators';
+
+/**
+ * @argument {requestActivity}
+ * @argument {onDeleteRequest}
+ */
+export default class PublicationsPublicationProofProofRequestedPanel extends Component {
+  @tracked isVerifyingDelete = false;
+  @tracked proofingActivity;
+
+  constructor() {
+    super(...arguments);
+    this.loadProofingActivity.perform();
+  }
+
+  get isProofingActivityFinished() {
+    return !isEmpty(this.proofingActivity.endDate);
+  }
+
+  @task
+  *loadProofingActivity() {
+    this.proofingActivity = yield this.args.requestActivity.proofingActivity;
```

Comment:
Het enige waar we die proofing-activity voor laden in een `@tracked` property is voor die end-date. Dat is volgens mij even duidelijk gewoon in de template via `@requestActivity.proofingActivity.endDate`. De template zal dan wel zorgen dat de nodige data geladen wordt.

Misschien is het wel nuttig om een `get isFinished()` toe te voegen aan het model van `proofing-activity` zodat de intentie iets meer duidelijk wordt via `@requestActivity.proofingActivity.isFinished`.